dags/utils/dataproc.py

```diff
@@ -530,3 +531,42 @@ def moz_dataproc_scriptrunner(parent_dag_name=None,
         create_dataproc_cluster >> run_script_on_dataproc >> delete_dataproc_cluster
         return dag
 # End moz_dataproc_scriptrunner
+
+
+def copy_artifacts_dev(dag, project_id, artifact_bucket, storage_bucket):
```

Comment:
I think when we're done with migration, we should move this bootstrap methods out of dags to some local scripts.
@acmiyaguchi what do you think?

Reply:
Yeah, it seems like a local script would be apt for this bootstrap piece. On the other-hand, this makes it obvious that there are some hidden dependencies when running this job locally. I kind of like seeing the dev dependencies in the local UI.